apps/chronicles/screens/ToastScreen.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { Button, EDSStyleSheet, Typography, useStyles } from "@equinor/mad-components";
+import { ScrollView, View } from "react-native";
+import { addToast } from "@equinor/mad-toast";
+import { useCoreStackNavigation } from "@equinor/mad-core";
+
+export default function ToastScreen() {
+    const styles = useStyles(themeStyles);
+    const navigation = useCoreStackNavigation();
+
+    return (
+        <ScrollView contentContainerStyle={styles.container}>
+            <View style={styles.readableContent}>
+                <Typography>
+                    Try out our different toast types by pressing the buttons below!
+                </Typography>
+                <View style={{ flexDirection: "row", gap: 16, marginTop: 8 }}>
+                    <Button
+                        title={"Success"}
+                        color="primary"
+                        onPress={() => addToast({ type: "SUCCESS", text: "Success!" })}
```

Comment:
Was the API for this made so that the transition from the old package is easier?
We have to opportunity to require devs to use a new format (like lowercased types). I don't know if the convenience of only replacing the import and not changing the types is worth it. They have to change code in all their files anyways and TS should complain as a guide. Thoughts?

Reply:
I agree that we have the opportunity to change the format, but I don't like changing the format if there is no clear benefit of doing so. I think people will appreciate it if we keep the transition as easy as possible, [which is why we made it a goal to keep the api as similar as possible.](https://github.com/equinor/mad/issues/521) Maybe @dansvend @HavardNot @Jonas-Omdal has any thoughts on this

Reply:
I understand. It is partially related to a comment I made below, but also comes from experience. Usually the enkleste er ofte det beste, and if there are two types of toasts floating around in the codebase (i.e. "ERROR" and "error"), then it quickly becomes confusing to work with.

Reply:
After looking into `react-native-salute`, it looks like they in fact use lowercase, and our apps' `addToastByType` use uppercase. I'll change our toast types to lowercase